containers/themes/ThemesSection.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import { c } from 'ttag';
+import {
+    useMailSettings,
+    useEventManager,
+    useApiWithoutResult,
+    SubTitle,
+    Alert,
+    ThemeCards,
+    ConfirmModal,
+    TextArea
+} from 'react-components';
+import { updateTheme } from 'proton-shared/lib/api/mailSettings';
+import { themeDark, themeLight, themeBlue, themeCustom } from './availableThemes.js';
+
+const availableThemes = [themeDark, themeLight, themeBlue, themeCustom];
```

Comment:
You should get the list from `proton-shared/lib/constants`:
```js
export const LIGHT_THEME = '/* light-theme */';
export const DARK_THEME = '/* dark-theme */';
export const BLUE_THEME = '/* blue-theme */';
export const THEMES = [LIGHT_THEME, DARK_THEME, BLUE_THEME];
```

Reply:
I followed a slightly different road with the last commit (0285b25)